RockLib.Messaging.Kafka/DependencyInjection/KafkaReceiverOptions.cs

```diff
@@ -6,37 +6,18 @@ namespace RockLib.Messaging.Kafka.DependencyInjection
     /// <summary>
     /// Defines the settings for creating instances of <see cref="KafkaReceiver"/>.
     /// </summary>
-    public class KafkaReceiverOptions
+    public class KafkaReceiverOptions : ConsumerConfig
     {
+        public KafkaReceiverOptions() : base()
+        {
+            EnableAutoOffsetStore = false;
+            AutoOffsetReset = Confluent.Kafka.AutoOffsetReset.Latest;
```

Comment:
NOTE: Because the `AutoOffsetReset` property is nullable, the compiler can't properly disambiguate it from the enum type. So, to refer to an enum value, we need to quality the type more.

https://stackoverflow.com/a/14386140